packages/webpack/src/plugins/generate-package-json-plugin.ts

```diff
@@ -34,22 +34,6 @@ export class GeneratePackageJsonPlugin implements WebpackPluginInstance {
             this.projectGraph
           );
 
-          const importHelpers = !!readTsConfig(this.options.tsConfig).options
```

Comment:
If this is removed, then package.json for apps might be missing the helper libs? I think we need to keep the feature intact since the eslint rule is not going to be wired up yet.

Reply:
This was actually always duplicate. The above `getHelperDependenciesFromProjectGraph` is already handling this.